GetRepoJSON/getRepoJSON.py

```diff
@@ -0,0 +1,52 @@
+# repos.py
+
+# A demo script that demonstrates api calls /user
+
+
+
+from time import sleep
+def delayserver():
+    sleep(0.01)
```

Comment:
😢 I assume this is here to deal with rate limiting?

Reply:
This is rate limiting. I could use suggestions. Whatever I put here, the customer is going to do too.

Reply:
Basically, you have one request executor that all requests go through.  It keeps track of how long it's been since the last request, how many requests are going simultaneously, etc., and it can do its own client-side rate limiting.  If you're under the limit, there's no need to wait, and everything can be faster.  I would have to think about it more since I don't know of an off-the-shelf library that does this.  Maybe it exists though.  I haven't needed it before.

Reply:
I would like to move the rate limiting to our python module to limit user error. https://github.com/AllSpiceIO/py-allspice/issues/6